### PR TITLE
test(Slider): add interaction tests for Slider component

### PR DESCRIPTION
**Pull request by niksa-monday**
- add first (initial) portion of interaction tests for Slider components
 - delete irrelevant unit tests (tests replaced by interaction tests)

### CODE REVIEW

File: src/components/Slider/__tests__/Slider.interactions.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import { within, userEvent } from "@storybook/testing-library";
+import { delay, interactionSuite, resetFocus } from "../../../__tests__/interactions-helper";
+
+// Decrease/Increase value by mouse click on Track/Rail of Slider
+const changeSliderValueByClickingOnTrackTest = async canvas => {
+  // prepare: take sizes of slider
+  const elRail = canvas.getByTestId("monday-slider-show-value-s__rail");
+  const rect = elRail.getBoundingClientRect();
+  // waiting for render
+  await delay(300);
```
Comment (laviomri) [inline src/components/Slider/__tests__/Slider.interactions.js:11]: consider using `waitFor` (from `@testing-library/react`) along with `expect(element).toBeVisible()`, like we did on [`waitForElementVisible`](https://github.com/mondaycom/monday-ui-react-core/blob/master/src/__tests__/interactions-helper.js#L99)

File: src/components/Slider/__tests__/Slider.interactions.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  const elThumb = within(elRail).getByRole("slider");
+  // go to start
+  userEvent.click(elRail, { clientX: Math.ceil(rect.left) });
+  await delay(300);
```
Comment (laviomri) [inline src/components/Slider/__tests__/Slider.interactions.js:15]: why do we need the delays (this one and the following ones) for?

**Review by laviomri — APPROVED** [review]
really cool! had some comments about the delays

File: src/components/Slider/__tests__/Slider.interactions.js (hunk 2 above)
Comment (niksa-monday) [reply src/components/Slider/__tests__/Slider.interactions.js:15]: Actually I just wanted to see it at storybook and forgot to remove.
Already removing...

Comment (niksa-monday) [reply src/components/Slider/__tests__/Slider.interactions.js]: Resolved

File: src/components/Slider/__tests__/Slider.interactions.js (hunk 1 above)
Comment (niksa-monday) [reply src/components/Slider/__tests__/Slider.interactions.js]: Changed to watiFor (i.e. to helper: waitForElementVisible from interactions-helper)

**Pull request merged**